### PR TITLE
Change Download Function

### PR DESCRIPTION
**Pull request by ArneGudermann**
Fix File download for Bones.

### CODE REVIEW

**Comment by sveneberth** [comment]
@ArneGudermann Your PR has a merge conflict. Please resolve it to be ready to review and merge.

File: vi/actions/file.py [hunk 1]
```diff
@@ -294,7 +294,8 @@ def enableViUnloadingWarning(self, *args, **kwargs ):
 
 	def doDownload(self, fileData):
 		a = html5.A()
-		a["href"] = "/file/download/%s/%s?download=1" % (fileData["dlkey"],fileData["name"])
+		a["download"] = fileData["name"]
+		a["href"] = "/file/download/%s/%s?download=1" % (fileData["dlkey"])
```
Comment (phorward) [inline vi/actions/file.py:298]: This format-string uses `%s` placeholder two times, but only one parameter is passed. Please use `f"..."`-strings here.

File: vi/widgets/file.py [hunk 2]
```diff
@@ -109,9 +109,10 @@ def download(self):
 		if not self.currentFile:
 			return
 
-		self.downloadA["href"] = self.currentFile["downloadUrl"]
-		self.downloadA["target"] = "_blank"
-		self.downloadA.element.click()
+		a = html5.A()
+		a["download"] = self.currentFile["name"]
+		a["href"] = "/file/download/%s/%s?download=1" % ( self.currentFile["dlkey"])
```
Comment (phorward) [inline vi/widgets/file.py:114]: This format-string uses `%s` placeholder two times, but only one parameter is passed. Please use `f"..."`-strings here.

**Review by phorward — CHANGES_REQUESTED** [review]
You Pull Request contains several major issues causing direct exception raise. It doesn't seem to be tested before. Please fix your code.

**Pull request closed without merging**